# **First, let's learn about our dataset, or the people who applied for credit cards...**

# **Need to reduce the number of digits in total income so it appears better on visualizations. We do this in two steps:  
Step 1:**

In [98]:
alter table dataset
add IncomeReduced decimal(7,3);


Commands completed successfully.

Total execution time: 00:00:00.117

# **Step 2:**

In [115]:

update dataset
set IncomeReduced = Total_income / 1000; 

(9709 rows affected)

Total execution time: 00:00:00.039

# _Now Total\_income has been reduced by $1,000._

# **Now we group applicants into income brackets of $20,000 and show what percentage from each group were approved for credit cards:**

In [118]:
-- Income bins ($20k) vs pct approved
with newTable as 
(select 
    ID,
    concat( 
        floor(IncomeReduced / 20) * 20, 
        '-', 
        floor(IncomeReduced / 20) * 20 + 10)
        as Income_group, 
    cast([Target] as int) as outcome
from dataset)

select 
    Income_group,
    100 *(sum(outcome)) / (count(ID)) as pct_Approved
from newTable 
group by Income_group
order by cast( SUBSTRING(Income_group, 1, CHARINDEX('-', Income_Group) - 1) as int);

(46 rows affected)

Total execution time: 00:00:00.025

Income_group pct_Approved 20-30 20 40-50 8 60-70 12 80-90 12 100-110 13 120-130 13 140-150 12 160-170 13 180-190 12 200-210 13 220-230 14 240-250 12 260-270 14 280-290 16 300-310 13 320-330 11 340-350 15 360-370 10 380-390 5 400-410 4 420-430 15 440-450 14 460-470 28 480-490 14 500-510 0 520-530 0 540-550 19 560-570 0 580-590 0 600-610 0 620-630 25 640-650 0 660-670 26 680-690 0 700-710 0 720-730 0 740-750 0 760-770 50 780-790 0 800-810 50 900-910 20 940-950 0 980-990 0 1120-1130 33 1340-1350 100 1560-1570 100

# _Income appears to be important in likelihood of being approved. 100% of all applicants in the top two income groups were approved._

# **Let's examine all approved applicants and compare their housing type.**

# _Above, the majority of approved applicants (88%) live in their owned house or apartment._

# **Next, let's compare the income type of all approved applicants.**

In [13]:
select 
    Income_type, 
    --count(ID) as totalApplied,
    --sum(cast([Target] as int)) as totalApproved,
    ( 100 * sum(cast([Target] as int)) / count(ID) ) as pctApproved
from dataset 
group by Income_type;

(5 rows affected)

Total execution time: 00:00:00.009

Income_type pctApproved State servant 12 Commercial associate 13 Pensioner 12 Working 13 Student 33

# _Students are much more likely to be approved (33%) than other applicants. However, only 3 students applied._